final_task/README.md

```diff
@@ -17,14 +17,21 @@ optional arguments:
   --json         Print result as JSON in stdout
   --verbose      Outputs verbose status messages
   --limit LIMIT  Limit news topics if this parameter provided
+  --date         Read cached news for provided URL. If "ALL" provided - prints all cached news for this date
```

Comment:
Please add a bit more info about how do you store cached news. What do you use and etc.

Reply:
Sure. Will fix it.